solr/core/src/java/org/apache/solr/search/QParser.java

```diff
@@ -360,6 +360,10 @@ public static QParser getParser(String qstr, String parserName, boolean allowLoc
     }
 
     QParserPlugin qplug = req.getCore().getQueryPlugin(parserName);
+    if (qplug == null){
+      // error: log ?
```

Comment:
I would log an error here, but `QParser` doesn't have a logger, should we add one?